indexer/packages/postgres/src/stores/pnl-ticks-table.ts

```diff
@@ -212,17 +212,24 @@ function convertPnlTicksFromDatabaseToPnlTicksCreateObject(
   return _.omit(pnlTicksFromDatabase, PnlTicksColumns.id);
 }
 
-export async function findLatestProcessedBlocktime(): Promise<string> {
+export async function findLatestProcessedBlocktimeAndCount(): Promise<{
+  maxBlockTime: string,
+  count: number,
```

Comment:
im a little confused, why are we counting the number or rows in `pnl_ticks`? How does that relate to "all subaccounts have been processed"

Reply:
This function is used to determine whether or not to run the pnl task.

Previously, we were looking at the last block time in the pnl table to determine whether or not to run the pnl task. This worked, but then we imposed a limit on the number of subaccounts/run to compute pnl for. What happened was we started to skip computing pnl on certain subaccounts because we were hitting the limit/run. So, this PR changes this check to see if the last run computed pnl for LIMIT # of subaccounts, and if so, proceed with the task & check the pnl timestamps of the subaccounts individually.

Reply:
https://github.com/dydxprotocol/v4-chain/blob/c26b4593a8ff5f03c4e205def779562ec5a4ecc0/indexer/services/roundtable/src/helpers/pnl-ticks-helper.ts#L82